hudi-client/hudi-java-client/src/test/java/org/apache/hudi/testutils/TestHoodieMetadataBase.java

```diff
@@ -303,9 +303,7 @@ protected HoodieWriteConfig.Builder getWriteConfigBuilder(HoodieFailedWritesClea
             .ignoreSpuriousDeletes(validateMetadataPayloadConsistency)
             .build())
         .withMetricsConfig(HoodieMetricsConfig.newBuilder().on(enableMetrics)
-            .withExecutorMetrics(enableMetrics).build())
-        .withMetricsGraphiteConfig(HoodieMetricsGraphiteConfig.newBuilder()
-            .usePrefix("unit-test").build())
+            .withExecutorMetrics(enableMetrics).withReporterType(MetricsReporterType.CONSOLE.name()).build())
```

Comment:
Do we actually use these metrics in the console output? the in memory option may be better suited for testing.

Reply:
sounds fair